### ISSUE TITLE
Unhandled Instruction: dadd

### ISSUE THREAD

**Issue by theballaam96** [N64Recomp/N64Recomp#90]
Game: Donkey Kong 64
Confirmed that the instruction is present in the function so it isn't an incorrect detection
![image](https://github.com/user-attachments/assets/b6c632af-c122-43b5-9412-ced833500e9b)

**Comment by theballaam96** [comment]
Synced my fork, rebuilt the exe and it's now having trouble recompiling a function it had no issue with before:
![image](https://github.com/user-attachments/assets/dbe9b999-bd54-4ec0-90a8-36ffab449fb2)

**Comment by Mr-Wiseguy** [comment]
If the recompiler isn't detecting a function that usually means either the function's symbol is marked as ABSOLUTE or that it has a size of zero in the elf. If you do `readelf -s` on the elf and grep for the symbol you should be able to see if it's in a proper section and has a nonzero size.

**Comment by Mr-Wiseguy** [comment]
To elaborate on that, if you're using splat and the function is handwritten asm or a GLOBAL_ASM, you'll need to add `asm_emit_size_directive: True` to your yaml and may also need to update asm processor so it can handle the new size directives.

**Comment by theballaam96** [comment]
Correct, it's listed in the elf file as having 0 size. I'll take a look into why

**Comment by Triticum0** [comment]
Update?  @theballaam96
